FlorenceBML/BMLor321/BMLor321.xml

```diff
@@ -34,8 +34,133 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <altIdentifier>
                             <idno>Marrassini ms. 21</idno>
                         </altIdentifier>
+                        <altIdentifier>
+                            <idno>Assemani CCCCV</idno>
+                        </altIdentifier>
                     </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1r" to="68v"/>
+                            <title>Gǝʿǝz-Latin Lexicon</title>
+                            <note>According to <bibl><ptr target="bm:bm:Assemani1719Bibliotheca"/><citedRange unit="page">431</citedRange>
+                                </bibl> the author is <persName ref="PRS14230GBRaimondi"/>.
+                                The Vocabulary is apparently taken from lyrics and arranged by initial letter, but many pages remain blank, i.e. the work is 
```

Comment:
I don't understand what "taken from lyrics" means

Reply:
I rendered to "poetry".

Reply:
It is still not clear (lemma definitions are rhymed?)

In any case this description is the description of work (study) record to be created, not of the manuscript (even if a work is contained in a unique copy, we still must keep content and carrier apart)

Reply:
Yes, I have removed this information to the STU record. It is not, as you expected a lexicon in a rhymed form, but Marrassini (or Assemani) suspected, that the lemmata are taken from poetic texts.